dockerfiles/e2e-saas/test_patches/Dashboard.patch

```diff
@@ -1,13 +1,15 @@
---- ./pageobjects/dashboard/Dashboard.ts	2020-02-27 16:38:40.167786069 +0100
-+++ ./pageobjects/dashboard/Dashboard2.ts	2020-02-27 16:39:10.067174761 +0100
```

Comment:
@Katka92 @ScrewTSW are we going to remove this patch for 7.11.0 ?

Reply:
You are commenting on an old file right now, but anyways, if the objects are removed from upstream in 7.11.0 we should be able to get rid of the patch, yes.

Reply:
@ScrewTSW could you please make sure that we will be not dependent on the UD menu - https://github.com/eclipse/che/issues/16165

Reply:
I can look into that next week